pipeline_dp/accumulator.py

```diff
@@ -174,6 +186,48 @@ def compute_metrics(self) -> float:
 
 
 @dataclass
+class VectorSummationParams:
+    noise_kind: NoiseKind
+    max_norm: float
+
+
+class VectorSummationAccumulator(Accumulator):
+    _vec_sum: np.ndarray
+
+    def __init__(self, params: VectorSummationParams,
+                 values: Iterable[Tuple[float]]) -> None:
```

Comment:
np.arrays are used very widely in Python and they are so much effective,  that it would be very ineffective for API clients which have data in np.arrays to convert to tuples. So I think it's worth to allow value to be not only tuple but np.array as well.

Reply:
Even though it's only a type hint and the users are free to use `np.ndarray` - I agree with that point. We should allow users (or even soft-requiring them) to use `np.ndarray`.

Reply:
Done